docs/reference/codelists.md

```diff
@@ -182,57 +182,57 @@ file: ../../schema/codelists/open/media_type.csv
 ---
 ```
 
-### roles
-
-The roles codelist is based on the [ISO19115 CI_RoleCode codelist](https://standards.iso.org/iso/19115/resources/Codelists/gml/CI_RoleCode.xml) with the addition of 'world_bank_team_lead' and the omission of codes covered by the `creator`, `contact_point` and `publisher` fields.
+### quantity_kind
```

Comment:
Explain the relationship to QUDT.

```suggestion
### quantity_kind

The quantity kind codelist is a subset of the codes in the [QUDT Quantity Kind Vocabulary](https://www.qudt.org/doc/DOC_VOCAB-QUANTITY-KINDS.html). It includes a code for the [currency quantity kind](https://qudt.org/vocab/quantitykind/Currency), which does not appear in the vocabulary because QUDT lists currencies in a separate graph from other quantity kinds.
```